worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java

```diff
@@ -873,6 +851,92 @@ public IChunkSet processSet(IChunk chunk, IChunkGet get, IChunkSet set) {
         return null;
     }
 
+    private boolean noBlacklistTrimXZ(
```

Comment:
I'm not sure about this method name. It might be best to move the `!(trimX || trimZ)` check into the parent method, and then having two separate methods for `trimChar` and `trimInt` as this also reduces the complexity of the overall code (albeit only be a single if statement)